examples/twodnavierstokes_stochasticforcing.jl

```diff
@@ -52,10 +52,10 @@ grid = TwoDGrid(dev, n, L)
 
 K = @. sqrt(grid.Krsq)
 forcing_spectrum = @. exp(-(K - forcing_wavenumber)^2 / (2 * forcing_bandwidth^2))
-forcing_spectrum[K .< ( 2 * 2π/L)] .= 0 # no power at low wavenumbers
-forcing_spectrum[K .> (20 * 2π/L)] .= 0 # no power at high wavenumbers
+@. forcing_spectrum = ifelse(K < 2, 0, forcing_spectrum)       # no power at low wavenumbers
```

Comment:
I see now (after reviewing your other PR) that you're removing the 2 pi/L factors from all the forcing terms in the examples. I guess that makes sense in terms of streamlining the code, since most of these domains have width 2 pi, but do you think it might confuse people if they adapt the domain size for their own simulations.

Reply:
That was accidental. I'm putting the `L/2π` factors back -- thanks for noticing!